test/service/test_manifest.py

```diff
@@ -1739,8 +1739,8 @@ def test_fetch_manifest(self, get_cached_manifest):
                 }
                 if format_ == ManifestFormat.curl:
                     expected_json['CommandLine'] = {
-                        'cmd.exe': f'curl.exe --location "{str(url)}" | curl.exe --config -',
-                        'bash': f"curl --location '{str(url)}' | curl --config -"
+                        'cmd.exe': f'curl.exe --location -f "{str(url)}" | curl.exe --config -',
```

Comment:
same

Reply:
Does [this comment](https://github.com/DataBiosphere/azul/pull/3183#discussion_r674303546) apply here?

Reply:
I vote yes, the comment applies.